### PR TITLE
小更新

### PR DESCRIPTION
**Pull request by Cortantse**

### CODE REVIEW

File: app.log
Comment (Cortantse) [inline app.log:1]: 新增日志文件

File: main.go [hunk 1]
```diff
@@ -99,32 +96,6 @@ func RegisterAdmin(mux *http.ServeMux) {
...
-	c := cron.New(cron.WithSeconds())
-
-	// 添加任务，每小时整点执行
-	_, err := c.AddFunc("0 0 0 * * *", printCurrentTime) // 6 个域
-	if err != nil {
-		fmt.Printf("添加任务时出错: %v\n", err)
-		return
-	}
-
-	// 启动 Cron 调度器
```
Comment (Cortantse) [inline main.go:114]: 放到专门的log.go

**Review by Cortantse — COMMENTED** [review]
self-check ok

**Review by GoronwySan — APPROVED** [review]
good good

**Pull request merged**